### ISSUE TITLE
Skills cap prematurely until you re-equip your weapon.

### ISSUE THREAD

**Issue by Aneyus** [EdenServer/community#56]
Example: My great katana just capped at level 27 as a level 12 SAM when it should have the A+ rank cap of 39. Even an F rank is still 29 at level 12. Re-equipping your weapon temporarily fixes the issue.
This bug was also on Kupo so I'm assuming it's a DSP issue.

**Comment by Wiggo32** [comment]
When you first equipped the weapon, do you know if you were at a lvl that would normally cap at 27, and then as you gained lvl's the cap just never adjusted. (which could indicate a packet is missing that needs to be sent on level up)

**Comment by Aneyus** [comment]
I was using a Tachi (Level 8 Great Katana) and A+ skill cap at level 8 is indeed 27. I zoned multiple times but I don't recall whether or not I changed jobs and/or unequiped the weapon between dinging level 8 and reaching 27 skill at level 12, but assuming I had gone straight from 8 to 12 I would have no reason to change my weapon.

tl;dr I probably equipped the weapon at level 8 which would indeed have a skill cap of 27

**Comment by roriffxi** [comment]
This is a common DSP bug, where you'll reach the cap that you can gain skill against the mob you are fighting and then DSP treats the skill as "capped" and prevents skillups until you blink your equipment to reset it.

**Comment by jmcmorris** [comment]
Does anyone know how to reproduce this? Thanks!

**Comment by beauxq** [comment]
I'm not sure about how to reproduce it, but I would try leveling up while under a level sync and then removing the level sync, then skilling up to see where it caps (all of this without ever changing any gear).

**Comment by archfiendrubicante** [comment]
https://gyazo.com/63772b5bb8a4f0c77e6ec92c4ecff638

This is probably the best way I've found to duplicate the issue: 
If you level up a basic character and equip a weapon it'll cap off far before the actual cap for that level.
In the above screenshot, the cap level for Sword should be 20 but displays as 7.

**Comment by archfiendrubicante** [comment]
https://gyazo.com/8db95e674adf96c7cd27fb6098a6f608

And an after shot, after re-equipping a sword.
